### PR TITLE
Fix value of seconds unit in example/benchmark-memory.php

### PR DESCRIPTION
**Pull request by tnsoftbear**
We expect second unit in $interval argument of EventLoop::repeat() function instead of millisecond. Script doesn't work as it should according to description in its comment. I've fixed that.

### CODE REVIEW

File: examples/benchmark-memory.php [hunk 1]
```diff
@@ -38,7 +38,7 @@
 EventLoop::repeat(1, function () use (&$runs) {
```
Comment (kelunik) [inline examples/benchmark-memory.php]: This should probably be 0.001 then?

Comment (tnsoftbear) [reply examples/benchmark-memory.php]: Probably yes, thank you. I've committed your suggested change.

File: examples/benchmark-memory.php [hunk 2]
```diff
@@ -25,20 +25,20 @@
...
         EventLoop::getDriver()->stop();
     });
 }
 
-EventLoop::repeat(1, function () use (&$runs) {
+EventLoop::repeat(0.001, function () use (&$runs) {
     $runs++;
 
-    EventLoop::repeat(1_000, function (string $watcher) {
+    EventLoop::repeat(0.001, function (string $watcher) {
```
Comment (kelunik) [inline examples/benchmark-memory.php]: This has been 1000ms so it should be 1s now.

Comment (tnsoftbear) [reply examples/benchmark-memory.php]: Thank you. I've fixed it to 1 second. Although, I didn't notice any difference in memory consumption or anything else.

**Review by kelunik — APPROVED** [review]

**Review by WyriHaximus — APPROVED** [review]

**Comment by WyriHaximus** [comment]
Thank you for spotting and fixing this 👍

**Pull request merged**